core/src/main/java/io/questdb/cairo/GenericRecordMetadata.java

```diff
@@ -88,6 +88,36 @@ public static RecordMetadata removeTimestamp(RecordMetadata that) {
         return that;
     }
 
+    public static RecordMetadata convertSymbolsToStrings(RecordMetadata that) {
+        boolean hasSymbol = false;
+        for (int i = 0, n = that.getColumnCount(); i < n; i++) {
+            int columnType = that.getColumnType(i);
+            if (ColumnType.isSymbol(columnType)) {
+                hasSymbol = true;
+                break;
+            }
+        }
+
+        if (!hasSymbol) {
+            return removeTimestamp(that);
+        }
+
+        GenericRecordMetadata metaCopy = new GenericRecordMetadata();
+        for (int i = 0, n = that.getColumnCount(); i < n; i++) {
+            int columnType = that.getColumnType(i);
+            metaCopy.add(new TableColumnMetadata(
+                    that.getColumnName(i),
+                    that.getColumnHash(i),
+                    ColumnType.isSymbol(columnType) ? ColumnType.STRING : columnType,
```

Comment:
Shouldn't 
that.isColumnIndexed(i) 
that.getIndexValueBlockCapacity(i)
be replaced with 
false, 
0 
if we're 'converting' from symbol to string ?